### ISSUE TITLE
Rooftop Storm (New Card)

### ISSUE THREAD

**Issue by kelsioboy** [magarena/magarena#1257]
name=Rooftop Storm
image=http://magiccards.info/scans/en/isd/71.jpg
value=3.877
rarity=R
type=Enchantment
cost={5}{U}
ability={0}: Cast a Zombie creature card from your hand without paying its mana cost. Activate this ability only if you have a creature card in your hand.
timing=enchantment
oracle=You may pay {0} rather than pay the mana cost for Zombie creature spells you cast.

**Comment by kelsioboy** [comment]
This Card has the condition:
Activate this ability only if you have a creature card in your hand.
But this condition is not perfect, to be perfect the condition should look like this:
Activate this ability only if you have a Zombie creature card in your hand.
What happens if this card is on a deck with only zombies in it, it is nice to play, bad on a mixed deck, not very good.
Unfortunately I can not increase the subtype for this condition. If you can do this, I think this card will look good on scrip, otherwise it is best to do it in a groove.

**Comment by kelsioboy** [comment]
Please disregard the comment above. The problem does not occur with the scrip I sent to you, I'm using a different one that had "You may cast" and the one I sent has not. But I'm sending another one to limit the "AI" from activating multiple times, as you did with other cards that have {0}.

**Comment by kelsioboy** [comment]
[Rooftop_Storm.txt](https://github.com/magarena/magarena/files/1226184/Rooftop_Storm.txt)

**Comment by melvinzhang** [comment]
@kelsioboy please edit your comment instead of attaching a new card. The activation limit is not needed as it prevents the AI from casting its whole hand of Zombie creatures. If I'm not wrong, when there is no Zombie creature card in hand, this ability cannot be activated, is that correct? If so both activation restrictions can be dropped.

**Comment by kelsioboy** [comment]
You're right the limitation of activating only when you have creature in your hand is really unnecessary, plus the limitation with the "AI" I think you should have, since the to select the same card in hand infinitely.
Of curiosity I did a test using "Rooftop Storm" with "Diregraf Colossus" in the field and I selected a same zombie card in my hand several times, the result is that the effect of "Diregraf Colossus" only created a token which is positive, Because it would be wrong to create more than one token with the same card (which was my curiosity). The negative part is that it has been stacked several times, so the limitation for "AI", because with it I think this can cause a loop, like other cards that have "{0}:" such as "Blessing of Leeches "that has this limitation to avoid this loop.
